### DISCUSSION TITLE
关于数字和单位的输入。

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#105] [Q&A]
你好，
输入数字：`\num{}`
输入数字和单位：`\SI{}{}`
有两点我不懂：

1. 我看siunitx红包说明，按理说我需要输入代码：`\usapackage{siunitx}`。为什么我没有添加`\usapackage{siunitx}`，tex文件仍旧可以编译呢？是不是模板默认加了很多宏包，那么我应该去哪里看哪些加了，哪些没有加呢？
2. `\num{}`在文字之后，很明显结果是数字特别挨着文字，所以应该使用：`~\num{}` 吗？

谢谢

**Comment by tanukihee** [comment]
> 是不是模板默认加了很多宏包，那么我应该去哪里看哪些加了，哪些没有加呢？

是，读文档

> `\num{}`在文字之后，很明显结果是数字特别- 挨着文字，所以应该使用：`~\num{}` 吗？

不是，普通的空格就可以，这个也在文档里说了

RTFM

**Reply by earthmoon** [reply]
> 是，读文档，最后一章
> 
> 小心收到 RTFM 的回复



> RTFM



> 是，读文档，最后一章
> 
> 小心收到 RTFM 的回复

找到了，谢谢！
